draft-ietf-ppm-dap.md

```diff
@@ -1217,15 +1222,14 @@ conveyed from the Leader to the Helper when initializing aggregation jobs
 
 ## Resource URLs
 
-DAP is defined in terms of "resources", such as reports ({{upload-flow}}),
+DAP is defined in terms of HTTP resources, such as reports ({{upload-flow}}),
 aggregation jobs ({{aggregate-flow}}), and collection jobs ({{collect-flow}}).
-Each resource has an associated URL. Resource URLs are specified by a sequence
-of string literals and variables. Variables are expanded into strings according
-to the following rules:
+Each resource has a URL. Resource URLs are specified as string literals
+containing variables. Variables are expanded into strings according to the
+following rules:
 
 * Variables `{leader}` and `{helper}` are replaced with the base API URL of the
-  Leader and Helper respectively (the base URLs are defined in
-  {{task-configuration}}).
+  Leader and Helper respectively.
```

Comment:
This is a bit of cleanup: "sequence of string literals" was misleading. I considered adding text about escaping `{` and `}` if they don't occur in a variable, but decided that we're not spelling out a general mechanism for variable expansion here. Rather we only spell out the expansion of five literals (`{leader}`, `{helper}`, `{task-id}`, `{aggregation-job-id}`, `{collection-job-id}`) and nothing else. If we want anything richer (or if something like httpdir or AD review asks for something better), then we should bite the bullet and do RFC 6570 templates, but I don't want to do something in between.